README.md

```diff
@@ -48,6 +48,33 @@ The following changes are a **VIOLATION** and **ARE NOT** included:
   * ❌ Re-engaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal release ❌ (was never included in MADS)
   * ❌ No disengaging on ACC MAIN in OFF state ❌
 
+⚒ Installation
+---
+
+### Comma URL (Comma 3 only quick setup)
+* 8.14 was last updated in July 2022, many changes have been made since then so test-c3 is recommended
+* Factory reset/uninstall previous software if you already installed a branch
+* Select Custom Software
+* Input the following URL: ```installer.comma.ai/sunnyhaibin/test-c3```
```

Comment:
Thanks for the review @Moodkiller !

@wabash1 Can we list these steps as numerical orders such as the one mentioned in "[Dynamic Lane Profile](https://github.com/sunnyhaibin/sunnypilot#dynamic-lane-profile-dlp)"?